test/va-letters/reducers/index.unit.spec.js

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+
+import lettersReducer from '../../../src/js/va-letters/reducers';
+
+const initialState = {
+  letters: [],
+  destination: {},
+  available: false
+};
+
+describe('letters reducer', () => {
+  it('should handle failure to fetch letters', () => {
+    const state = lettersReducer.letters(
```

Comment:
The only thing I'm struggling to understand is the import of the reducers why the import is returning something like this: 
```
lettersReducer: {
  letters: function()
}
```
I would think the import would just import the single reducer function we need here and not an object with the reducer function within it.